ci/local-run.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+## This file is potentially useful for doing a clean environment build on MacOS ##
+## This is my personal way of running the build, YMMV - jsimms                  ##
+
+set -euo pipefail
+IFS=$'\n\t'
+
+die() { echo "fatal: $*" >&2; exit 1; }
+
+TOPLEVEL=$(git -C "$(cd "$(dirname "$0")" >/dev/null || exit 1; pwd)" rev-parse --show-toplevel) || die "TOPLEVEL fail"
+
+cd "$TOPLEVEL"
+
+TEMP="$(mktemp -d -t TEMP.XXXXXXX)" || die "failed to make tmpdir"
+cleanup() { [[ -n "${TEMP:-}" ]] && rm -rf "${TEMP}"; }
+trap cleanup EXIT
+
+BUILD_PATH=(
+  "$HOME/.cargo/bin"
+  "/opt/twitter_mde/bin"
```

Comment:
This leaks Twitter-specific environment variables. What's the binary dependencies on these folders and can we replace them by something already set in environment?

Reply:
unnecessary, i'll remove it